src/ffi/index.ts

```diff
@@ -1,9 +1,10 @@
-import bindings = require('bindings');
+import path from 'path';
 import logger, { DEFAULT_LOG_LEVEL } from '../logger';
 import { LogLevel } from '../logger/types';
 import { Ffi } from './types';
 
-const ffiLib: Ffi = bindings('pact.node');
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const ffiLib: Ffi = require('node-gyp-build')(path.join(__dirname, '..', '..'));
```

Comment:
Can this be done without disabling `no-var-requires`?

Reply:
yes probably! this actually wants wrapping in a method so that if for whatever reason we can load the libs we can catch it and present a nice message to the user.

sorry we can’t support for platform/arch at the moment, it was x / y, why not raise an issue and xyz

Reply:
yes now typescript and eslint friendly :)